run_in_parallel.py

```diff
@@ -45,21 +52,30 @@ def parse_arguments():
     Parses commandline arguments
     :return: Parsed arguments
     """
-    parser = argparse.ArgumentParser('Run behave in parallel mode for scenarios')
-    parser.add_argument('--command_line_args', '-a', help='Command line arguments', default=DEFAULT_COMMAND_LINE_ARGS)
+    parser = argparse.ArgumentParser('Run behave in sequential mode for scenarios')
+    parser.add_argument('--stop_on_failure', '-s', help='Stop on test failure', default=DEFAULT_STOP_ON_FAILURE)
+    parser.add_argument('--show_skipped_tests', '-k', help='Show skipped tests', default=DEFAULT_SHOW_SKIPPED_TESTS)
+    parser.add_argument('--log_level', '-l', help='Logging level', default=DEFAULT_LOG_LEVEL)
     parser.add_argument('--format', '-f', help='Behave format', default=DEFAULT_BEHAVE_FORMAT)
-    parser.add_argument('--acceptance_features_directory', '-d', help='specify directory containing features',
-                        default=DEFAULT_FEATURES_DIRECTORY)
-    parser.add_argument('--processes', '-p', type=int,
-                        help=f'Maximum number of processes. Default [{DEFAULT_PROCESSES}] ', default=DEFAULT_PROCESSES)
-    parser.add_argument('--test_tags', '-t', help='specify behave tags to run', default=DEFAULT_TAGS)
+    parser.add_argument('--test_tags', '-t', help='Behave tags to run', default=DEFAULT_TAGS)
+    parser.add_argument('--feature_directory', '-fd', help='Feature directory', default=DEFAULT_FEATURE_DIRECTORY)
+    parser.add_argument('--processes', '-p', type=int, help=f'Maximum number of processes. '
+                                                            f'Default [{DEFAULT_PROCESSES}] ',
+                        default=DEFAULT_PROCESSES)
     parser.add_argument('--timeout', '-tout', type=int,
                         help='Maximum seconds to execute each scenario. Default = 300', default=300)
 
     args = parser.parse_args()
 
-    # Handle Tag ANDing properly
-    args.tags = create_behave_tags(args.test_tags)
+    # Create behave command line arguments
+    args.behave_args = {
```

Comment:
This is holding both the arguments for the dry run to select scenarios and the actual tests runs, it would be better to split them out, then you can pass the whole object into the relevant function later rather than picking the keys out.

Reply:
fixed